test/app/controllers/integration_examples_controller.rb

```diff
@@ -16,6 +16,9 @@ def controller_inline_baseline
   end
 
   def controller_to_string
+    # ensure render_to_string isn't broken by monkey patch
+    render_to_string("integration_examples/_controller_inline", locals: { message: "bar" })
```

Comment:
I'm not sure I understand what this code does. Can you explain a little

Reply:
@jonspalmer I've added a comment 😄